app/cmd_manager/cmd_manager.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+
+from app.book import Book
+from app.cmd_manager.display.manager import DisplayManager
+from app.cmd_manager.print.manager import PrintManager
+from app.cmd_manager.serialize.manager import SerializeManager
+from app.validators import CommandValidatorMixin
+
+
+class CommandManager(CommandValidatorMixin):
+    def __init__(self, command: str, method_type: str) -> None:
+        self.command = command
+        self.method_type = method_type
+
+    def execute_command(self, book: Book) -> Any:
+        self.validate_command(self.command)
+
+        if self.command == "display":
+            displayer = DisplayManager().set_displayer(self.method_type)
+            displayer.display(book)
+
+        elif self.command == "print":
+            printer = PrintManager().set_printer(self.method_type)
+            printer.print_book(book)
+
+        elif self.command == "serialize":
+            serializer = SerializeManager().set_serializer(self.method_type)
+            return serializer.serialize(book)
```

Comment:
The CommandManager class has to be modified every time a new command type is added, which violates the OCP. Ideally, the class should be open for extension but closed for modification.
refactor:

Command Interface: Create a common interface for all commands. This allows new commands to be added without modifying existing code.
Factory Pattern: Use a factory pattern to create command objects. This separates the instantiation of command objects from their execution.
Command Objects: Implement each command as a separate class that adheres to the command interface. This makes adding new commands easier and your code more modular.